cmd/kubeops/main.go

```diff
@@ -90,12 +90,19 @@ func main() {
 	// Routes
 	// Auth not necessary here with Helm 3 because it's done by Kubernetes.
 	addRoute := handler.AddRouteWith(r.PathPrefix("/v1").Subrouter(), withHandlerConfig)
+	// Deprecate non-cluster-aware URIs.
 	addRoute("GET", "/releases", handler.ListAllReleases)
 	addRoute("GET", "/namespaces/{namespace}/releases", handler.ListReleases)
 	addRoute("POST", "/namespaces/{namespace}/releases", handler.CreateRelease)
 	addRoute("GET", "/namespaces/{namespace}/releases/{releaseName}", handler.GetRelease)
 	addRoute("PUT", "/namespaces/{namespace}/releases/{releaseName}", handler.OperateRelease)
 	addRoute("DELETE", "/namespaces/{namespace}/releases/{releaseName}", handler.DeleteRelease)
```

Comment:
I guess it's better to do add a redirect in the nginx conf rather than maintaining here both set of routes. Can we do that?

Reply:
We could add redirect(s) in the nginx conf, but I don't know that it's better in this case. Here I'm ensuring they still work while the frontend has not been updated, but will remove them once the frontend always sends the cluster name. I won't be keeping them beyond that.